multi-arch/samples.md

```diff
@@ -0,0 +1,157 @@
+
+### Non-Goals
+
+We do not take on porting existing samples to non-x86 architectures.
+
+## Proposal
+
+The samples operator config object already supports the specification of multiple 
+architectures.
+
+It currently requires that only a single architecture is specified, and that 
+it is x86_64.
+
+In conjunction, the import of content from https://github.com/openshift/library supports 
+downloading multiple architectures, and segregating multiple architectures.  And 
+the tagging in https://github.com/openshift/library allows providers to designate which
+samples go to which platforms.
+
+So enabling the above to allow PPC and Z content if it is available is item one.
+
+But as previously stated, there is currently no such content, so the samples operator
+should do nothing when it comes up in Managed state, and report Available=true, Progressing=false,
+Degraged=false.
+
+We also need to ensure that marking the samples operator as Unmanaged or Removed 
+
+There are some interesting details and caveats that will be discussed in the 
+implementation details section.
+
+### User Stories [optional]
+
+#### Story 1
+
+#### Story 2
+
+### Implementation Details/Notes/Constraints [optional]
+
+In a somewhat related note, there have been a lot of side conversations regarding the Jenkins imagestream
+in particular, as the openshift/jenkins images are included in the 4.1/4.2 payloads.
+
+Clayton weighed in with Ben Parees and they concluded that unless/until the openshift/jenkins images 
+are fully removed from the install payload (i.e. the devtools team, say as part of the Jenkins operator work, 
+somehow facilitates the removal of the openshift/jenkins images from the payload), the x86 openshift/jenkins 
+images will be utilized as part of building the non-x86 install payload.
+
+### Risks and Mitigations
+
+Unexpected outcomes with respect to the overall multi-arch design.
+
+Unexpected pushback from IBM on the lack of non-x86 samples.
+
+## Design Details
+
+### Test Plan
+
+### Graduation Criteria
+
+#### Examples
+
+##### Dev Preview -> Tech Preview
+
+##### Tech Preview -> GA 
+
+##### Removing a deprecated feature
+
+### Upgrade / Downgrade Strategy
+
+No caveats or issues.  Standard, existing approaches apply.
+
+### Version Skew Strategy
+
+No caveats or issues.  Standard, existing approaches apply.
+
+## Implementation History
+
+## Drawbacks
+
+## Alternatives
+
+Remove samples operator as a CVO operator.  Make it a Day 2, Day X operation, where all of the existing samples 
+today are installed via OLM Operator Hub.  The selections on the Hub can then explicitly note whether or not they
+provide non-x86 options.
+
+## Infrastructure Needed [optional]
+
+New PR e2e jobs running on non-x86, or hybrid topologies, where clear means for the tests to determine the 
+architectures employed such that common tests can be run across multiple architectures.
```

Comment:
I hear that something in this space will happen, but am not yet clear on the details.